src/server/index.js

```diff
@@ -1,10 +1,26 @@
 const express = require('express');
+const bodyParser = require('body-parser');
+const mongoose = require('mongoose');
+
+const item = require('./routes/item.route');
```

Comment:
I think that we don't need to duplicate 'route' part, so please name files just:
```
./routes/item
./routes/items
```